languages/fsharp/exercises/concept/values/.docs/after.md

```diff
@@ -0,0 +1,15 @@
+In F#, a _value_ is everything that has a type and can be defined. That includes booleans, integers, strings, lists, but also functions.
```

Comment:
Does F# have objects? Are they values?

Reply:
Yes, objects are types that can be defined and are thus values.

Reply:
So I assume that with "objects" you mean classes, and yes they are also values. I purposely did not mention them though in the list, as classes are not used that much in F# and I'd rather not mention records already.